packages/components/card/src/base-card/BaseCard.tsx

```diff
@@ -0,0 +1,354 @@
+import { cx, css } from 'emotion';
+import React, { forwardRef, useCallback, useState } from 'react';
+import type {
+  AnchorHTMLAttributes,
+  FocusEventHandler,
+  KeyboardEventHandler,
+  MouseEvent,
+  MouseEventHandler,
+  ReactElement,
+  ReactNode,
+} from 'react';
+import tokens from '@contentful/f36-tokens';
+import { Box, Flex } from '@contentful/f36-core';
+import type {
+  CommonProps,
+  MarginProps,
+  PolymorphicComponent,
+  PolymorphicProps,
+} from '@contentful/f36-core';
+import { Icon, IconComponent } from '@contentful/f36-icon';
+import type { ButtonProps } from '@contentful/f36-button';
+import { DragHandle } from '@contentful/f36-drag-handle';
+import type { ObjectInterpolation } from '@emotion/serialize';
+
+import { getBaseCardStyles } from './BaseCard.styles';
+import { CardActions } from './CardActions';
+
+export const DEFAULT_TAG = 'article';
+
+const generateCardStyles = ({
+  hasHeader,
+  isFocused,
+  isHovered,
+  isSelected,
+}: {
+  hasHeader: boolean;
+  isFocused: boolean;
+  isHovered: boolean;
+  isSelected: boolean;
+}) => {
+  const styles: ObjectInterpolation<undefined> = {};
+
+  if (!hasHeader) {
+    styles.paddingTop = tokens.spacingM;
+  }
+
+  if (isHovered) {
+    styles.borderColor = tokens.colorPrimary;
+    styles.cursor = 'pointer';
+  }
+
+  if (isFocused) {
+    styles.borderColor = tokens.colorPrimary;
+    styles.boxShadow = tokens.glowPrimary;
+    styles.outline = 0;
+  }
+
+  if (isSelected) {
+    styles.backgroundColor = tokens.blue100;
+    styles.borderColor = tokens.colorPrimary;
+    styles.boxShadow = tokens.glowPrimary;
+  }
+
+  return css(styles);
+};
+
+export type CardElement = 'a' | 'article' | 'button' | 'div';
+
+type AnchorProps =
```

Comment:
I think these properties can be automatically recognized when you use `as="a"`. I'm not sure if you need to specify this type here.